.github/workflows/push.yml

```
@@ -44,17 +44,23 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}-${{ matrix.tennens }}:${{ github.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
-  dispatch:
+  branch_name:
     runs-on: ubuntu-latest
-    needs: 
-      - build_and_push_nuxt
+    outputs:
+      branch: ${{ steps.extract_branch.outputs.branch }}
     steps:
-      - uses: actions/checkout@v2
       - name: Extract branch name
         shell: bash
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
         id: extract_branch
+
+  dispatch:
+    runs-on: ubuntu-latest
+    needs:
+      - build_and_push_nuxt
```

In this workflow file the buld and push step is called build_and_push_nuxt
```suggestion
      - build_and_push_nuxt
```